ttnn/cpp/ttnn/operations/eltwise/binary/binary_composite.hpp

```diff
@@ -254,5 +270,11 @@ constexpr auto outer = ttnn::register_operation_with_auto_launch_op<
 constexpr auto polyval = ttnn::register_operation_with_auto_launch_op<
     "ttnn::polyval",
     operations::binary::ExecuteBinaryCompositeOpsPolyval<operations::binary::BinaryCompositeOpType::POLYVAL>>();
+constexpr auto gcd = ttnn::register_operation_with_auto_launch_op<
+    "ttnn::gcd",
+    operations::binary::ExecuteGCD>();
```

Comment:
Are we certain we need auto formatting here?

Reply:
> Are we certain we need auto formatting here?

Since all binary composite operations have been using auto-formatting, I also used it for the GCD and LCM implementations.

Reply:
Please try and remove it and see if the tests can pass.

I don't want new operations using the auto format at all.